web/src/ApiButton.tsx

```diff
@@ -274,7 +275,9 @@ export function ApiButton(props: React.PropsWithChildren<ApiButtonProps>) {
   const { className, uiButton, ...buttonProps } = props
 
   const [loading, setLoading] = useState(false)
-  const [inputValues, setInputValues] = useState(new Map<string, any>())
+  const [inputValues, setInputValues] = usePersistentState<{
```

Comment:
it turns out `JSON.stringify` on a `Map` just returns `"{}"` :(

Reply:
(there isn't a 😭 reaction)